diff --git a/pyproject.toml b/pyproject.toml
index 3283fce..89d3e8c 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -45,7 +45,6 @@ python-multipart = "^0.0.5"
 sanic = {version = ">=20.12.2,<22.0.0", optional = true}
 aiohttp = {version = "^3.7.4.post0", optional = true}
 fastapi = {version = ">=0.65.2", optional = true}
-"backports.cached-property" = "^1.0.1"

 [tool.poetry.dev-dependencies]
 pytest = "^7.1"
diff --git a/strawberry/field.py b/strawberry/field.py
index 4e7ee4b..7272188 100644
--- a/strawberry/field.py
+++ b/strawberry/field.py
@@ -18,7 +18,6 @@ from typing import (
     overload,
 )

-from backports.cached_property import cached_property
 from typing_extensions import Literal

 from strawberry.annotation import StrawberryAnnotation
diff --git a/strawberry/types/fields/resolver.py b/strawberry/types/fields/resolver.py
index 0409233..be97594 100644
--- a/strawberry/types/fields/resolver.py
+++ b/strawberry/types/fields/resolver.py
@@ -6,8 +6,6 @@ import sys
 from inspect import isasyncgenfunction, iscoroutinefunction
 from typing import Callable, Dict, Generic, List, Mapping, Optional, TypeVar, Union

-from backports.cached_property import cached_property
-
 from strawberry.annotation import StrawberryAnnotation
 from strawberry.arguments import StrawberryArgument
 from strawberry.exceptions import MissingArgumentsAnnotationsError
diff --git a/strawberry/types/info.py b/strawberry/types/info.py
index a172c04..6bbb578 100644
--- a/strawberry/types/info.py
+++ b/strawberry/types/info.py
@@ -2,8 +2,6 @@ import dataclasses
 import warnings
 from typing import TYPE_CHECKING, Any, Dict, Generic, List, Optional, TypeVar, Union

-from backports.cached_property import cached_property
-
 from graphql import GraphQLResolveInfo, OperationDefinitionNode
 from graphql.language import FieldNode
 from graphql.pyutils.path import Path
